api/filter/schema.go

```diff
@@ -106,6 +106,18 @@ func parseCert(crt, key string) ([]string, error) {
 }
 
 func handleSpecialField(resource string, reqBody []byte) ([]byte, error) {
+	var bodyMap map[string]interface{}
+	err := json.Unmarshal(reqBody, &bodyMap)
+	if err != nil {
+		return reqBody, fmt.Errorf("read request body failed: %s", err)
+	}
+	if _, ok := bodyMap["create_time"]; ok {
+		return reqBody, fmt.Errorf("we don't accept create_time from client")
```

Comment:
Here we don't add any parameters, just use errors.New is fine.

Reply:
If need to format your error message using `fmt.Errorf()`.
If we do not need format using `errors.New()`.

Reply:
fixed.